### ISSUE TITLE
Search: Search doesn't show results on master

### ISSUE THREAD

**Issue by bryphe** [onivim/oni#2487]
**Oni Version:** 0.3.7 (`master`)
**Operating System:** Windows

**Issue:**

Search functionality doesn't seem to be working on master. When I run a search over Oni, I get a quick spinner in the sidebar and then no results show up.

**Steps to reproduce:**

- Build `master` build of Oni
- Open the search pane with `Control+Shift+f`
- Type `'Neovim'`

Expected: Should see search results
Actual: No search results are displayed

**Comment by bryphe** [comment]
It looks like this code that was responsible for it in `src/Services/Search/index.ts` was removed:
```
        query.onSearchCompleted.subscribe(result => {
            this._onSearchCompleted.dispatch()
            const visualizer = new QuickFixSearchResultsViewer(this._editorManager)	                this._onSearchCompleted.dispatch()
            visualizer.showResult(result)
```
As part of this commit of the new find-in-files functionality.

@TalAmuyal - should we have an alternate way to bubble up search results in this case? Or does it make sense to just remove the search pane and pivot to your new strategy on master?

**Comment by bryphe** [comment]
(cc @jonasws)

**Comment by TalAmuyal** [comment]
Assigned myself.

**Comment by TalAmuyal** [comment]
Should be fixed now in `master` ([see #2489](https://github.com/onivim/oni/pull/2489)).
